### PR TITLE
[FIX] l10n_fr_pos_cert : error while deleting a journal created by POS

### PR DESCRIPTION
**Pull request by abla001**
whith France localization, deleting a journal created by POS
leads to two errors.

first one is just a typing error.

second one is an infinite loop caused by calling super().unlink()
from methods marked with the api.ondelete

opw-2750852

### CODE REVIEW

**Comment by abla001** [comment]
@simongoffin

**Comment by jco-odoo** [comment]
@abla001 This can not be good.  You should not delete the unlink to super.  The api.ondelete is used to maybe block when you uninstall the module.

**Comment by abla001** [comment]
> @abla001 This can not be good. You should not delete the unlink to super. The api.ondelete is used to maybe block when you uninstall the module.

the super().unlink() is causing an infinite loop because the function tagged with api.ondelete so it will be called when we call super().unlink() and so on.

Compared to other models the only function that calls super().unlink() is unlink() to override it, other unlink functions are there to block or rise a user error (like in this case).

File: addons/l10n_fr_pos_cert/models/account_bank_statement.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ def _unlink_except_created_by_pos(self):
                 continue
             if statement.state != 'open':
                 raise UserError(_('You cannot modify anything on a bank statement (name: %s) that was created by point of sale operations.') % statement.name)
-        return super().unlink()
+
```
Comment (jco-odoo) [inline addons/l10n_fr_pos_cert/models/account_bank_statement.py:18]: no extra empty line needed

**Comment by jco-odoo** [comment]
@simongoffin otherwise lgtm

**Review by simongoffin — COMMENTED** [review]
Yes it is legit

**Review by simongoffin — APPROVED** [review]

**Pull request closed without merging**